docs/source/pillars/computing_technology.md

```diff
@@ -84,11 +84,16 @@ This {term}`application component <application component>` is the tools used by
   - This may include shared file storage, databases, collaborative writing, and other web applications.
     This must only be shared amongst users within the same project.
   - Optional
-* - 2.1.8.
-  - Your TRE could include licenced commercial software if required by researchers, but additional risks must be recorded and mitigated where neccesary.
-  - For example, if an application must connect to an external licensing server, you must be confident that only licensing information is sent to this server, and that any network connections are secure.
-  - Optional
+* - 2.1.8
+  - Your TRE must ensure that any shared services are only available to users working on the same project.
+  - Your TRE might not provide shared services.
+  - Mandatory (if applicable)
```

Comment:
```suggestion
  - Mandatory
```

I think the if applicable is clear given we say "any shared services". I think we have done similar elsewhere.